Update Novelsat_NS1000.md (#886)
SkylineCommunications/dataminer-docs-connectors: connector/doc/Moxa_NPort_5000_Series_Technical.md

- **General**: System description, object ID, uptime, contact, name, location, number of services.
- **Interfaces**: Physical and virtual interface statistics.
- **Network**: Datagram, address translation, and discard statistics.
- **TCP**: Table of all active TCP connections and session-level metrics.
- **UDP**: UDP datagram counters, errors, and traffic stats.

> - **Network Translation**: he Address Translation tables contain the NetworkAddress to physical address equivalences.

Reviewer: Something seems to have gone wrong here. Is this what you meant?
```suggestion
- **Network Translation**: The Address Translation tables show which network address corresponds with which physical address.
```
